raiden/tests/integration/network/transport/rif_comms/test_client.py

```diff
@@ -22,23 +21,6 @@ def comms_nodes(amount_of_nodes) -> {int, CommsNode}:
         node.stop()
 
 
-@pytest.mark.parametrize("amount_of_nodes", [1])
-def test_locate_own_peer_id(comms_nodes):
-    comms_node = comms_nodes[1]
-
-    # attempt to locate self
-    assert comms_node.client._get_peer_id(comms_node.address) is not ""
-
-
-# FIXME: causes ERR_NO_PEERS_IN_ROUTING_TABLE in comms node although it passes
-@pytest.mark.parametrize("amount_of_nodes", [1])
-def test_locate_unregistered_peer_id(comms_nodes):
-    comms_node = comms_nodes[1]
```

Comment:
same here, i know it was failing, but why removed?

Reply:
same as previous reply